index.html

```diff
@@ -139,6 +140,17 @@ <h3>
           is invalid or no last packet of a key frame has been received yet,
           <code>contentType</code> is missing.
         </p>
+      </dd>
+      <dt>
+        <dfn><code>jitterBufferTargetDelay</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          This value is increased by the target jitter buffer delay every time a
+          sample is emitted by the jitter buffer. The added target is the target
+          delay, in seconds, at the time that the sample was emitted from the
+          jitter buffer.
```

Comment:
Should add "To get the average target delay, divide by (other number)" - I think it is jitterBufferEmittedCount.

for jitterBufferDelay, we have the words "The average jitter buffer delay can be calculated by dividing the [jitterBufferDelay](https://www.w3.org/TR/webrtc-stats/#dom-rtcinboundrtpstreamstats-jitterbufferdelay) with the [jitterBufferEmittedCount](https://www.w3.org/TR/webrtc-stats/#dom-rtcinboundrtpstreamstats-jitterbufferemittedcount)."

Reply:
Done!